package.json

```diff
@@ -11,47 +11,75 @@
     "umd"
   ],
   "scripts": {
-    "build": "nwb build-react-component",
-    "clean": "nwb clean-module && nwb clean-demo",
-    "lint": "npx eslint ./",
+    "build": "npm run build:umd && npm run build:demo",
+    "build:demo": "NODE_ENV=development webpack --mode=development",
+    "build:umd": "NODE_ENV=production webpack --mode=production",
+    "build:es": "mkdir -p es && cp -r src/* es && NODE_ENV=production MODULE_FORMAT=es npx babel es -d es",
+    "build:cjs": "mkdir -p lib && cp -r src/* lib && NODE_ENV=production MODULE_FORMAT=cjs npx babel lib -d lib",
+    "parcel": "parcel demo/src/index.html",
```

Comment:
1. This PR  ProjectMirador/mirador-image-tools#49 and the main Mirador repo use webpack instead of parcel to serve the content. While I don't have a strong preference about this and we deferred changes like introducing vite, I think we should try to use consistent approaches across the different repos. Is there a particular reason for using parcel here, even though webpack is already used for the build scripts?

2. Similar thing about script names: it would be nice to be able to start every application with `npm run start`.